integration-tests/tests/billing-projects.js

```diff
@@ -139,11 +149,32 @@ const setAjaxMockValues = async (testPage, ownedBillingProjectName, notOwnedBill
     email: 'testuser2@example.com', role: 'Owner'
   }]
 
+  const listWorkspacesResult = [
```

Comment:
This data is for testing the workspace list on a Google project and an Azure project (previously the workspace list was always empty).